src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSMultiBatchSink.java

```diff
@@ -137,7 +136,7 @@ public void prepareRun(BatchSinkContext context) throws IOException, Instantiati
       outputProperties.putAll(RecordFilterOutputFormat.configure(validatingOutputFormat.getOutputFormatClassName(),
                                                                  config.splitField, name, schema));
       outputProperties.put(FileOutputFormat.OUTDIR, config.getOutputDir(context.getLogicalStartTime(), name));
-
+      outputProperties.put(GCSBatchSink.CONTENT_TYPE, config.getContentType());
```

Comment:
does multi sink need same validation for format and content type?

Reply:
Yes. We need the same validation for format and content type in GCSMultiBatchSink as well. This is already in place.

Reply:
I see.